src/components/ScaleBuilder.vue

```diff
@@ -349,6 +342,18 @@ async function doImport(importerKey: ImporterKey, event: Event) {
           />
           <span>A5</span>
         </div>
+
+        <!--<div class="control">
```

Comment:
Commented out code should state the reason for it's existence or be removed completely.

Reply:
Have added comment